README.md

```diff
@@ -102,6 +102,13 @@ environment variables:
 
 - `TEST_VERBOSITY`: Increase logging output for tests. Default: `0`.
 
+- `ENABLE_RETRIES`: Enables retrying requests if a test does not succeed.
+   Defaults: `true`.
```

Comment:
I wonder if, to preserve existing behavior, we should default this to `false` and then liberally apply `true` on the tests we know would benefit?

Reply:
I think for:
* tests that are already working enabling retries will not have any effect
* tests that already exist and are properly broken, enabling retries will make the test run a tiny bit longer (since the request gets retried) but still fail
* tests that already exist and are flaky, enabling retries will maybe resolve the flakiness

I can see how that might cause unexpected behaviour for a user, so will flip it back to default to `false`